src/imitation/scripts/train_preference_comparisons.py

```diff
@@ -155,22 +157,43 @@ def train_preference_comparisons(
             relabel_reward_fn=relabel_reward_fn,
         )
 
-    if trajectory_path is None:
+    if num_agents < 1:
+        raise ValueError("num_agents must be at least 1!")
+
+    def make_agent_trainer(seed: Optional[int] = None):
```

Comment:
Now lines 151-158 aren't being used?

Reply:
Good catch! I guess there was a merge here that slipped by me.